Error compiling alumet-agent: the trait `vergen_lib::entries::Add` is not implemented for `Gitcl`

**Describe the bug**

I have an error compiling Alumet-agent since a few days

**Environment information**
OS: Debian 13
Alumet version²: latest

**To Reproduce**
`rustup update stable`

 `cargo install --git https://github.com/alumet-dev/alumet.git alumet-agent'`

**Logs / Output**
```
   [...]
   Compiling idna_adapter v1.2.1                    
   Compiling idna v1.1.0                         
   Compiling alumet-agent v0.9.2 (/root/.cargo/git/checkouts/alumet-41f6c4ff5183d397/e5d9a70/agent)      
   Compiling url v2.5.8                                                                            
error[E0277]: the trait bound `Gitcl: vergen_lib::entries::Add` is not satisfied                   
   --> agent/build.rs:21:27                                                                        
    |                                                                                              
 21 |         .add_instructions(&git)?
    |                           ^^^^ the trait `vergen_lib::entries::Add` is not implemented for `Gitcl`
    |                                            
help: trait impl with same name found                                                              
   --> /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/vergen-gitcl-1.0.8/src/gitcl/mod.rs:910:1
    |                                                                                              
910 | impl AddEntries for Gitcl {                                                                  
    | ^^^^^^^^^^^^^^^^^^^^^^^^^                                                                    
    = note: perhaps two different versions of crate `vergen_lib` are being used?                   
note: there are multiple different versions of crate `vergen_lib` in the dependency graph          
   --> /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/vergen-lib-9.1.0/src/entries.rs:45:1
    |                                                                                              
 45 | pub trait Add {                            
    | ^^^^^^^^^^^^^ this is the required trait                                                     
    |   
   ::: agent/build.rs:3:5                           
    |                                               
  3 | use vergen::{BuildBuilder, CargoBuilder, Emitter, RustcBuilder};                                   
    |     ------ one version of crate `vergen_lib` used here, as a dependency of crate `vergen`          
  4 | use vergen_gitcl::GitclBuilder;               
    |     ------------ one version of crate `vergen_lib` used here, as a dependency of crate `vergen_gitcl`
    |                                               
   ::: /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/vergen-gitcl-1.0.8/src/gitcl/mod.rs:236:1
    |                                               
236 | pub struct Gitcl {                            
    | ---------------- this type doesn't implement the required trait                                    
    |                                               
   ::: /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/vergen-lib-0.1.6/src/entries.rs:45:1
    |                                               
 45 | pub trait Add {                               
    | ------------- this is the found trait         
    = help: you can use `cargo tree` to explore your dependency tree                                     
    = note: required for the cast from `&Gitcl` to `&dyn vergen_lib::entries::Add`                       

For more information about this error, try `rustc --explain E0277`.                                      
error: could not compile `alumet-agent` (build script) due to 1 previous error                           
warning: build failed, waiting for other jobs to finish...                                               
error: failed to compile `alumet-agent v0.9.2 (https://github.com/alumet-dev/alumet.git#e5d9a707)`, intermediate artifacts can be found at `/tmp/cargo-installrw5r31`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/agent/Cargo.toml
+++ b/agent/Cargo.toml
@@ -67,9 +67,10 @@ pretty_assertions.workspace = true
 tempfile.workspace = true
 
 [build-dependencies]
-vergen = { version = "9.0.4", default-features = false, features = [
+vergen = { version = "=9.0.6", default-features = false, features = [
     "build",
     "cargo",
     "rustc",
 ] }
-vergen-gitcl = "1.0.5"
+vergen-gitcl = "=1.0.8"
+vergen-lib = "=0.1.6"

